index.bs

```diff
@@ -1104,6 +1112,29 @@ a minimum of friction, e.g., the user will not have to dig around in their pocke
 example of the latter, when the user is accessing the [=[RP]=] from a given client for the first time, they may be required to
 use a [=roaming authenticator=] which was originally registered with the [=[RP]=] using a different client.
 
+### User Verification Protection enumeration (enum <dfn enum>UserVerificationProtection</dfn>) ### {#user-verification-protection}
+
+<pre class="idl">
+    enum MatcherProtection {
```

Comment:
Seeing you call this "matcher protection" reinforces my belief that we should rename "user verification" to something like "user matching" or "person matching" since it's all about ensuring that the person getting an assertion matches the person who created the key pair.